django-reference-app/django-ref-config/settings.py

```diff
@@ -1,125 +0,0 @@
-"""
```

Comment:
Is this change an error? Deleting the contents of this file will break everything.

Reply:
ok, but to add postgres to the app you need to make some changes to `settings.py`, so should you not merge or rebase from master and make changes on this branch?